docs/docs/cli-reference/engine-start.md

```diff
@@ -14,7 +14,8 @@ This command will do nothing if the Kurtosis engine is already running.
 You may optionally pass in the following flags with this command:
 * `--log-level`: The level that the started engine should log at. Options include: `panic`, `fatal`, `error`, `warning`, `info`, `debug`, or `trace`. The engine logs at the `info` level by default.
 * `--version`: The version (Docker tag) of the Kurtosis engine that should be started. If not set, the engine will start up with the default version.
+* `--author`:  "The author (Docker username) of the Kurtosis engine that should be started (blank will start the kurtosistech version). The same author and version are used for the API Container & Files Artifact Expander.
```

Comment:
Does the same version also be used? or is it only the author? What happens if the user sets a custom version with the `--api-container-version` in the `kurtosis enclave add` command?